recipe/meta.yaml

```diff
@@ -57,7 +59,7 @@ outputs:
 
 about:
   home: http://golang.org
-  license: BSD-Style
+  license: BSD-3-Clause
```

Comment:
GitHub recognizes [the license]( https://github.com/golang/go/blob/go1.4/LICENSE ) as BSD-3-Clause